lib/index.js

```diff
@@ -284,9 +276,9 @@ Server.prototype = {
       } else {
         apimlTlsOptions = this.webServer.getTlsOptions();
       }
-      installLogger.info('The http port given to the APIML is: ', webAppOptions.httpPort);
-      installLogger.info('The https port given to the APIML is: ', webAppOptions.httpsPort);
-      installLogger.info('The zlux-apiml config are: ', apimlConfig);
+      installLogger.info('ZWED0033I', webAppOptions.httpPort); //installLogger.info('The http port given to the APIML is: ', webAppOptions.httpPort);
```

Comment:
Same thought here: is 3 ids really appropriate?

Reply:
Addressed.